### PR TITLE
feat: add zotero json output support

### PR DESCRIPTION
**Pull request by musayann**

### CODE REVIEW

File: src/components/cards/Node.tsx [hunk 1]
```diff
@@ -22,17 +21,17 @@ export function NodeCard({ data, sourcePosition, targetPosition, selected }: any
...
   return (
     <div className="group cursor-pointer">
-      <Handle type="target" position={targetPosition} id={data.Key} />
+      <Handle type="target" position={targetPosition} id={data.key} />
         <div onClick={onClick} className={className}>
           <div className="relative">
-            <p className="z-10">{data.Title}</p>
-            <p className="z-10">{data.Author}</p>
+            <p className="z-10">{data.title}</p>
+            <p className="z-10">{data.authors}</p>
```
Comment (NNesta) [inline src/components/cards/Node.tsx:34]: Can you do conditional rendering for both data.title and data.authors

File: src/components/cards/Node.tsx [hunk 2]
```diff
@@ -42,23 +41,23 @@ export function NodeCard({ data, sourcePosition, targetPosition, selected }: any
...
                 </div>
               )}
               <p>
-                <strong>Title</strong>: {data.Title}
+                <strong>Title</strong>: {data.title}
               </p>
               <p>
-                <strong>Tags</strong>: {data.Manual_Tags}
+                <strong>Tags</strong>: {data.tags}
               </p>
```
Comment (NNesta) [inline src/components/cards/Node.tsx:56]: Can you also add condition rendering for data.title and data.authors

**Review by NNesta — CHANGES_REQUESTED** [review]

File: src/data/case-studies/index.ts [hunk 3]
```diff
@@ -1,7 +1,5 @@
 import { Group } from "@/types/interfaces";
-import output from "../../../public/data.json";
-import { technologies } from "./technologies";
-import { levels } from "./levels";
+import output from "@/data/zotero";
```
Comment (kareraolivier) [inline src/data/case-studies/index.ts:2]: This block of code here 
```jsx 
const toolKits = output.filter(
  (data) => data?.PARSED_MANUAL_TAGS["CORE_TOOLKIT"]
);
```
Is updated to: 
```jsx
const toolKits = output.filter(
  (data) => data?.PARSED_MANUAL_TAGS["CORE TOOLKIT"]
);
```

**Review by kareraolivier — COMMENTED** [review]
We are also not using `generate-output-json.js`

File: src/data/data-resources.ts [hunk 4]
```diff
@@ -1,9 +1,5 @@
...
-
-const dataResources = output
-  .map((entry) => entry?.PARSED_MANUAL_TAGS?.ACCESS)
-  .flat()
-  .filter((resource) => resource && !resource.includes("Open Access"))
-  .filter((data) => data) as string[];
-
-export const filteredResources = Array.from(new Set(dataResources));
+export const dataResources = [
+  "Open Source",
```
Comment (musayann) [inline src/data/data-resources.ts:2]: why are we hardcoding these values ?

**Pull request merged**